script/Ciel/Day 4/04_01_CIEL04_4_2.txt

```diff
@@ -1,37 +1,40 @@
 [sha:f21616ab0366d75979f7381dfeca3e5daad62caa]{
 -- Page 94, Offset 8410.
 -- ……いや、いくらそんな目をされても頷けない。
-...No matter how much she looks at me with those eyes, I can't agree.
+No matter how much she looks at me with those eyes, I can't agree.
 }
 [sha:34ec7c37d9746ea6965c4c43adad452fda8f7b8d]{
 -- Page 94, Offset 8411.
 -- バラバラにされても生き返る女だの、黒い豹だの燃える死骸だのといった非常識を、相手にできる筈がない。
-There is no way I could deal with the absurdity of a woman who could come back to life after being torn apart, or a black panther, or burning corpses.
+A woman who can come back to life even after being torn to shreds, black panthers, burning corpses...
+ All of this absurd nonsense is way above my pay grade.
```

Comment:
Earlier today my grandfather asked me if I had ever heard the expression "above my pay grade". I think he might not think very highly of my education